2-mandatory/2-water-bottle.js

```diff
@@ -15,22 +15,31 @@ You have to implement the missing features according to the specification.
 */
 
 // Here is your starting point:
+
 let bottle = {
   volume: 0,
   fillUp: function () {
-    // calling this function should completely fill your bottle (volume = 100);
+    this.volume = 100;
   },
   pour: function () {
-    // calling this function should increase your bottle volume by 10 units;
+    if (this.volume + 10 > 100) {
+      this.volume = 100;
+    } else {
+      this.volume += 10;
+    }
```

Comment:
using a conditional expression (`... ? ... : ...`) you could write this in a single line